src/DataObjectAnnotator.php

```diff
@@ -70,31 +75,44 @@ public function __construct()
      */
     protected function setEnabledClasses($supportedParentClass)
     {
-        foreach((array)ClassInfo::subclassesFor($supportedParentClass) as $class)
-        {
+        foreach ((array)ClassInfo::subclassesFor($supportedParentClass) as $class) {
             $classInfo = new AnnotateClassInfo($class);
-            if($this->permissionChecker->moduleIsAllowed($classInfo->getModuleName())) {
+            if ($this->permissionChecker->moduleIsAllowed($classInfo->getModuleName())) {
                 $this->annotatableClasses[$class] = $classInfo->getClassFilePath();
             }
         }
     }
 
     /**
-     * @param $moduleName
-     * @return array
+     * @return boolean
      */
-    public function getClassesForModule($moduleName)
+    public static function isEnabled()
     {
-        $classes = array();
+        return (bool)static::config()->get('enabled');
```

Comment:
Let's declare this if it's a config property:

```php
/** 
 * Whether annotations are enabled
 *
 * @config
 * @var bool
 */
private static $enabled = true; // ?
```

Reply:
It is a config already, but as it's called quite a few times, it's just a single point instead of having the full static::config etc. to write out.